src/wmnds/components/form-elements/dropdown/_dropdown.njk

```diff
@@ -2,22 +2,54 @@
 
 {# Imports #}
 {% from "wmnds/components/form-elements/label/_label.njk" import wmndsLabel %}
+{% from "wmnds/components/form-elements/error-message/_error-message.njk" import wmndsFeErrorMessage %}
 
 {# Params #}
-{% set id = params.id if params.id else "dropdown" %}
-<div class="wmnds-fe-group">
-  <div class="wmnds-fe-dropdown">
+{% set id = params.id if params.id else "dropdown-example" %}
+{% set name = params.name if params.name else "dropdown-example" %}
+{% set items = params.items if params.items %}
+{% set classes = " " + params.classes if params.classes %}
+{% set describedBy = params.describedBy if params.describedBy %}
+{# Error Message Params #}
+{% set groupErrorClass = " wmnds-fe-group--error" if params.error %}
+{% set errorContentText = params.errorMessage.contentText if params.errorMessage.contentText else "Please select an option" %}
+{% set errorContentHTML = params.errorMessage.contentHTML if params.errorMessage.contentHTML else null %}
+{% set errorClasses = " " + params.errorMessage.classes if params.errorMessage.classes else null %}
+{% set errorId = params.errorMessage.id if params.errorMessage.id else null %}
+
+<div class="wmnds-fe-group{{groupErrorClass}}">
+  <div class="wmnds-fe-dropdown{{classes}}">
+    {% if params.error -%}
+      {{
+        wmndsFeErrorMessage(
+          {
+            contentText: errorContentText,
+            contentHTML: errorContentHTML,
+            classes: errorClasses,
+            id: errorId
+          }
+        )
+      }}
+    {% endif -%}
     {{
       wmndsLabel({
-        contentText: 'Dropdown label',
+        contentText: params.label,
         for: id
-      }) | trim
+      })
     }}
-    <select class="wmnds-fe-dropdown__select" id="{{id}}" name="{{id}}">
-      <option value="" selected="selected">Choose from list</option>
-      {%- for i in range(1,6) %}
+    <select class="wmnds-fe-dropdown__select" id="{{id}}" name="{{name}}" {%if describedBy%}aria-describedBy="{{describedBy}}"{%endif%}>
```

Comment:
Dropdown must have the individual errorclass not sure if there is one for select, but on input it is wmnds-fe-input--error. It should have a red border if there's an error.

Reply:
There isn't a class for it in the css and I checked zeplin but there are no error styles for dropdown so maybe its best to leave this for a separate issue?